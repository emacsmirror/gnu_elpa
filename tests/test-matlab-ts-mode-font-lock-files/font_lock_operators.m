% -*- matlab-ts -*-
aa = inf;

x = 1 + 2 - 3 * 4 / 5

a = [1 2 3; 4 5 6];
b = [2 2 2; 1 1 1];
c = a .* b

d = a .\ b

e = a .^ 2

f = [1 2; 3 4] ^ 2

g = a.'
h = a'

j = 1 == 2

k = 1 ~= 2

l = 3 > 3
m = 3 >= 3
n = 3 < 3
o = 3 <= 3

b1 = true & false;
b2 = true | false;
b3 = true && false;
b4 = true || false;
b5 = ~b4;

sqr = @(x) x .^ 2;

mc = ?MyClass;

