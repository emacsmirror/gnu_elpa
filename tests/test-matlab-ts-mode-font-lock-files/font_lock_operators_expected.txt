c ccc ccccccccc ccc
vv o BBBD

v o n o n o n o n o n

v o bn n nD n n nbD
v o bn n nD n n nbD
v o d oo d

v o d oo d

v o d oo n

v o bn nD n nb o n

v o doo
v o do

V o n oo n

v o n oo n

v o n o n
v o n oo n
v o n o n
v o n oo n

vv o BBBB o BBBBBD
vv o BBBB o BBBBBD
vv o BBBB oo BBBBBD
vv o BBBB oo BBBBBD
vv o oddD

vvv o obdb d oo nD

vv o odddddddD

